Sourcery/Models/TypeName.swift

```diff
@@ -78,24 +79,22 @@ public final class TypeName: NSObject, AutoCoding, AutoEquatable, AutoDiffable,
                     unwrappedTypeName = name.drop(first: "ImplicitlyUnwrappedOptional<".characters.count, last: 1)
                 }
                 self.unwrappedTypeName = unwrappedTypeName.bracketsBalancing()
+                self.isGeneric = unwrappedTypeName.contains("<") && unwrappedTypeName.characters.last == ">"
```

Comment:
I guess if you move this after `else` branch you can do this just once using `unwrappedTypeName`